src/main/java/mate/academy/util/DatabaseConnector.java

```diff
@@ -0,0 +1,26 @@
+package mate.academy.util;
+
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class DatabaseConnector {
+    private static final String CONNECTION_URL = "jdbc:mysql://localhost:3306/mate_jdbc?serverTimezone=UTC";
+    private static final Properties PROPERTIES;
+
+    static {
+        PROPERTIES = new Properties();
+        PROPERTIES.put("user", "root");
+        PROPERTIES.put("password", "root");
```

Comment:
Consider externalizing the database credentials (username and password) to a configuration file or environment variables for better security and flexibility. Hardcoding credentials in the source code is not recommended for production environments.